test/e2e/utils.go

```diff
@@ -215,3 +233,24 @@ func checkImageRemoved(ctx context.Context, t *testing.T, nodes []string, images
 		t.Error("not all nodes cleaned")
 	}
 }
+
+func dockerPullImage(image string) (string, error) {
+	args := []string{"pull", image}
+	cmd := exec.Command("docker", args...)
+	stdoutStderr, err := cmd.CombinedOutput()
+	return strings.TrimSpace(string(stdoutStderr)), err
```

Comment:
Can you inject `stdoutStderr` into these error messages

e.g. 

```suggestion
	if err != nil {
		err = fmt.Errorf("%w: %s", err, stdoutStderr)
	}
	return strings.TrimSpace(string(stdoutStderr)), err
```

Otherwise we just end up with something like `fork/exec: exit status 1` as the error message.

Reply:
Changes made here and everywhere else I could find the pattern